style.css

```diff
@@ -262,8 +262,46 @@ h4.upper-h4 {
         font-size: 5rem;
     }
 
-    .card:nth-child(3) {
-        display: none;
+    .cards {
+        width: 70vh;
+        max-height: 80vw;
```

Comment:
using relative units **vh for width** and **vw for height** could lead to unexpected responsive behaviour for different screen sizes. 

Try use **vw for width** and **vh for height** instead.

Reply:
The height and width are in correspondence to the property transform: rotate(-90deg);
Since the div has been rotated to achieve the horizontal scroll of content with vertical scroll of our device as stated in the [codepen](https://codepen.io/meduzen/pen/vGpwMB) you mentioned.

Reply:
Yes, the codepen demonstrates this. LGTM :)

Reply:
As discussed above, the .cards class has some unexpected responsive behaviour for different screen sizes. Please refer to issue #14  for context, and kindly fix. Thanks :)